src/components/fields/money-field/money-field.form.story.js

```diff
@@ -14,129 +15,137 @@ import Readme from './README.md';
 import MoneyField from './money-field';
 import MoneyInput from '../../inputs/money-input';
 
-const formToDoc = values => ({
-  price: MoneyInput.convertToMoneyValue(values.price),
-  pricePerTon: MoneyInput.convertToMoneyValue(values.pricePerTon),
-  discountedPrice: MoneyInput.convertToMoneyValue(values.discountedPrice),
+const formToDoc = (values, locale) => ({
+  price: MoneyInput.convertToMoneyValue(values.price, locale),
+  pricePerTon: MoneyInput.convertToMoneyValue(values.pricePerTon, locale),
+  discountedPrice: MoneyInput.convertToMoneyValue(
+    values.discountedPrice,
+    locale
+  ),
 });
 
-storiesOf('Examples|Forms/Fields', module)
-  .addDecorator(withKnobs)
-  .addDecorator(withReadme(Readme))
-  .add('MoneyField', () => {
-    const currencies = ['EUR', 'USD', 'AED', 'KWD'];
-    const horizontalConstraint = select(
-      'horizontalConstraint',
-      ['m', 'l', 'xl', 'scale'],
-      'm'
-    );
-    return (
-      <Section>
-        <Formik
-          initialValues={{
-            price: MoneyInput.parseMoneyValue(),
-            pricePerTon: MoneyInput.parseMoneyValue(),
-            discountedPrice: MoneyInput.parseMoneyValue(),
-          }}
-          validate={values => {
-            const errors = { price: {}, pricePerTon: {}, discountedPrice: {} };
-            if (MoneyInput.isEmpty(values.price)) errors.price.missing = true;
-            else if (MoneyInput.isHighPrecision(values.price))
-              errors.price.unsupportedHighPrecision = true;
+const Story = injectIntl(props => {
```

Comment:
I moved the story to be a variable to injectItl, will point out the real changes